app/controllers/hours_controller.rb

```diff
@@ -3,16 +3,13 @@ class HoursController < ApplicationController
   before_action :set_volunteer
 
   def index
-    hour = Hour.new(volunteer: @volunteer)
-    authorize hour
-    @hours = Hour.where(volunteer: @volunteer)
+    authorize @volunteer.hours.first || Hour
```

Comment:
Not sure why loading the first record is needed, can we just always do `authorize Hour`?

Reply:
if I remove `@volunteer.hours.first`then I got undefined method error
![screen shot 2017-10-11 at 17 35 03](https://user-images.githubusercontent.com/11731325/31450750-8a605ec4-aeaa-11e7-818a-a76fa3ac2347.png)

Reply:
we decided to leave it for the time being as it is now, since it is running and refactor the policy next week